### PR TITLE
feat: Upgrade to angular 13

### PR DESCRIPTION
**Pull request by rickynguyen4590**
Upgrade  lib to angular 13 with Ivy, I think we should change package version 1.8 cause it's a major changes.

### CODE REVIEW

File: projects/line-truncation-lib/package.json [hunk 1]
```diff
@@ -21,8 +21,8 @@
     "url": "https://github.com/DiZhou92/ngx-line-truncation"
   },
   "peerDependencies": {
-    "@angular/common": "^10.0.0 || ^11.0.0 || ^12.0.0",
-    "@angular/core": "^10.0.0 || ^11.0.0 || ^12.0.0"
+    "@angular/common": "^13.0.0",
```
Comment (diz517) [inline projects/line-truncation-lib/package.json:24]: can you make those peer deps support v10, v11 and v12 still, 
e.g. 
```
     "@angular/common": "^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0",
```
at least i know there are people who needs v10 supports

Comment (diz517) [reply projects/line-truncation-lib/package.json:24]: see https://github.com/diz517/ngx-line-truncation/issues/47

**Review by diz517 — APPROVED** [review]
Thanks for making the pr, please hold on to merge, still I need to verify that its working against testing instance https://github.com/diz517/ngx-line-truncation-demo

Once its good to go, i will merge and publish the package.

File: projects/line-truncation-lib/package.json (hunk 1 above)
Comment (rickynguyen4590) [reply projects/line-truncation-lib/package.json]: @diz517  Updated to keep all major versions already

**Pull request merged**